src/jsonrpc/miner_jsonrpc_info.erl

```diff
@@ -78,6 +78,9 @@ handle_rpc(<<"info_summary">>, []) ->
         firmware_version => FirmwareVersion,
         gateway_details => GWInfo
     };
+handle_rpc(<<"info_version">>, []) ->
+    #{version => miner:version()};
```

Comment:
On second review.. why not just add it to the info_summary as `miner_version` in

Reply:
bump @lthiery

Reply:
I'm not sure why, yet, but the `info_summary` calls take 2-3 seconds to return most of the time. The response for `miner:version()` is milliseconds.  I like it split into it's own call, but can definitely see the benefits of reducing the api surface and putting into into the summary call.  I do want to be able to get the version over rpc though.